src/OpenLoco/src/GameCommands/GameCommands.h

```diff
@@ -1549,6 +1549,35 @@ namespace OpenLoco::GameCommands
         return doCommand(GameCommand::changeCompanyFace, regs) != FAILURE;
     }
 
+    struct ClearLandArgs
+    {
+        static constexpr auto command = GameCommand::clearLand;
+        ClearLandArgs() = default;
+        explicit ClearLandArgs(const registers& regs)
+            : centre(regs.ax, regs.cx)
+            , pointA(regs.edx & 0xFFFF, regs.ebp & 0xFFFF)
+            , pointB(regs.edx >> 16, regs.ebp >> 16)
+            , flags(regs.bl)
+        {
+        }
+
+        World::Pos2 centre;
+        World::Pos2 pointA;
+        World::Pos2 pointB;
+        uint8_t flags;
```

Comment:
in general i've not been putting the flags in these structs. They get passed into the function as a separate thing.